packages/components/src/utils/checkbox-radio-button-wrapper-utils.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { addChangeListener, changeHandler } from './checkbox-radio-button-wrapper-utils';
+
+describe('changeHandler()', () => {
+  it('should use correct selector for radio-button-wrapper and set display = none and immediately set it back to default value', () => {
```

Comment:
To be precise, I think this util got nothing todo with any wrapper or component.
So the correct wording should mention `input type="radio". 
Please also adjust the checkbox test accordingly.